meeting-notes/2018/2018-08-20--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,204 @@
+     - Review part of the documentation (https://github.com/whyrusleeping/gx/issues?q=is%3Aissue+is%3Aopen+label%3Adocumentation).
+   - Encourage some community contribution.
+   - In progress: Badger integrity (`go-ipfs`: #5213, #5280) and memory (#5177, #5390) issues.
+   	- - Notes from @bigs:
+     	 - When Commit(nil) returns (as we have it implemeneted in go-ds-badger), entries have been committed to the WAL at least and should be truncate-safe.
+ - Next:
+   - Ease the use of `gx-go link` when dependencies don't match, e.g., developing in `go-unixfs` and testing in `go-ipfs` (https://github.com/whyrusleeping/gx-go/issues/46).
+   - Keep working on `gx` docs and code clarity.
+   - Incorporate the feedback of the DAG traversal PR (https://github.com/ipfs/go-ipfs/pull/5257) and split it in the corresponding repos (`go-unixfs` and `go-ipld-format`). PR under review.
+
+@stebalien
+ - Done:
+   - IPLD:
+     - thinking through go-ipld interfaces from a performance perspective (no concrete progress)
+     - have a plan to fix the interfaces and move to the IPLD org: https://github.com/ipld/go-ipld/issues/2
+   - fix (in review): stop go-libp2p from NAT port mapping non-nat (e.g., localhost) addresses.
+   - fix (in review): stop go-ipfs from listening on random ports.
+   - misc fixes, reviews, gx
+ - Next:
+   - multistream-select (again)
+   - continue work on IPLD interfaces/migration.
+ - Blocked:
+   - @daviddias: [Remove bitswap unwant](https://github.com/ipfs/go-ipfs/pull/5308)
+   - @daviddias: [New multiaddr protocol](https://github.com/multiformats/multiaddr/pull/68)
+ - Ask:
+   - Please review PRs!
+   - Bug me if you're blocked on something.
+
+@Magik6k
+ - Done:
+   - Coverage from Jenkins - https://github.com/ipfs/go-ipfs/pull/5389
+     - Needs a review + some admin actions
+   - Rebasing / finishing PRs
+ - Blocked:
+ - Next:
+   - get go-cidutil to be released on gx for coreapi/dht - https://github.com/ipfs/go-ipfs/pull/4804
+     - + update the PR to use it
+   - More coreapi command refactoring
+   - `ipfs name resolve --stream` (@stebalien says ❤️)
+   - workaround peerHost.SetStreamHandler not returning errors by using SetStreamHandlerMatch instead (for `ipfs p2p` refactor)
+
+@keks
+ - Done:
+   - cmds refactor is on it's way
+     - development branch of go-ipfs now uses newest version of refactored cmds lib
+     - sharness tests fail
+   - start of discussion on gx-gomod
+     - https://github.com/whyrusleeping/gx/issues/200
+ - Blocking:
+   - keep gx-gxmod discussion alive (not urgent on my side)
+   - get https://github.com/ipfs/go-ipfs/pull/5035 reviewed (not *super* urgent)
+ - Next:
+   - fix sharness tests
+   - continue designing gx-gomod
+
+@hsanjuan
+ - Done:
+   - Another round. Refs -r: https://github.com/ipfs/go-ipfs/pull/5337
+   - Extract MFS (waiting OK): https://github.com/ipfs/go-ipfs/pull/5391
+ 
+ - Blocked:
+ - Next:
+ 
+@bigs
```

Comment:
Did you get to meet with @jacobheun and @travisperson? Ref https://github.com/ipfs/pm/pull/678/files#r210003491